### ISSUE TITLE
Picture disappears

### ISSUE THREAD

**Issue by DursunCanPoyraz** [phenomlab/fancybox#1]
I click on the picture. I click on an empty space on the site. The picture disappears.

```
$ php flarum info
Flarum core 0.1.0-beta.15
PHP version: 7.4.12
Loaded extensions: Core, date, libxml, openssl, pcre, zlib, bz2, calendar, ctype, curl, hash, filter, ftp, gettext, gmp, SPL, iconv, pcntl, readline, Reflection,session, standard, shmop, SimpleXML, mbstring, tokenizer, xml, i360, bcmath, dom, fileinfo, gd, imagick, imap, intl, json, exif, mysqli, mysqlnd, PDFlib, PDO, pdo_mysql, pdo_sqlite, Phar, posix, soap, sockets, sqlite3, tidy, xmlreader, xmlrpc, xmlwriter, xsl, zip, ionCube Loader, Zend OPcache
+---------------------------------+------------------+--------+
| Flarum Extensions               |                  |        |
+---------------------------------+------------------+--------+
| ID                              | Version          | Commit |
+---------------------------------+------------------+--------+
| flarum-approval                 | v0.1.0-beta.15   |        |
| flarum-bbcode                   | v0.1.0-beta.15   |        |
| flarum-emoji                    | v0.1.0-beta.15   |        |
| flarum-lang-english             | v0.1.0-beta.15   |        |
| flarum-flags                    | v0.1.0-beta.15   |        |
| flarum-likes                    | v0.1.0-beta.15   |        |
| flarum-lock                     | v0.1.0-beta.15   |        |
| flarum-markdown                 | v0.1.0-beta.15   |        |
| flarum-mentions                 | v0.1.0-beta.15   |        |
| flarum-statistics               | v0.1.0-beta.15   |        |
| flarum-sticky                   | v0.1.0-beta.15.1 |        |
| flarum-subscriptions            | v0.1.0-beta.15   |        |
| flarum-suspend                  | v0.1.0-beta.15   |        |
| flarum-tags                     | v0.1.0-beta.15   |        |
| tolgaaaltas-lang-turkish        | 0.15.5           |        |
| tolgaaaltas-turkish             | 0.15.5           |        |
| michaelbelgium-discussion-views | v5.0.0           |        |
| fof-upload                      | 0.12.9           |        |
| phenomlab-fancybox              | 0.1.2            |        |
+---------------------------------+------------------+--------+
Base URL: https://beta15.quzz.net
Installation path: /home/oyundede/beta15.quzz.net
Debug mode: off
```

https://beta15.quzz.net/d/2-test-1/3

**Comment by phenomlab** [comment]
I'm aware of this and working on a fix

**Comment by phenomlab** [comment]
@DursunCanPoyraz resolved with version 0.1.3
